src/gui/qjacktrip.cpp

```diff
@@ -1186,9 +1144,217 @@ void QJackTrip::loadSettings()
+        }
 
+        m_ui->patchServerCheckBox->setChecked(cliSettings->getPatchServerAudio());
+        m_ui->upmixCheckBox->setChecked(cliSettings->getPatchServerAudio());
+        m_ui->zeroCheckBox->setChecked(cliSettings->getUnderrunMode() == JackTrip::ZEROS);
+        m_ui->timeoutCheckBox->setChecked(cliSettings->getStopOnTimeout());
+        m_ui->clientNameEdit->setText(cliSettings->getClientName());
+        m_ui->remoteNameEdit->setText(cliSettings->getRemoteClientName());
+        m_ui->localPortSpinBox->setValue(cliSettings->getBindPort());
+        m_ui->remotePortSpinBox->setValue(cliSettings->getPeerPort());
+        int basePort = cliSettings->getServerUdpPort();
+        if (basePort == 0) {
+            // TODO: This currently mirrors the behaviour seen in UdpHubListener.cpp, but
+            // I'm not sure it's particularly intuitive. It makes sense if the bind port
+            // was changed using the offset flag -o but not if it was changed using -B.
+            // These two cases are not currently distinguished between.
+            basePort = 61002 + cliSettings->getBindPort() - gDefaultPort;
+        }
+        m_ui->basePortSpinBox->setValue(basePort);
+        int queueLength = cliSettings->getQueueLength();
+        m_ui->queueLengthSpinBox->setValue(
+            queueLength > 0
+                ? queueLength
+                : settings.value(QStringLiteral("QueueLength"), gDefaultQueueLength)
+                      .toInt());
+        m_ui->redundancySpinBox->setValue(cliSettings->getRedundancy());
+        AudioInterface::audioBitResolutionT resolution =
+            cliSettings->getAudioBitResolution();
+        if (resolution == AudioInterface::BIT8) {
+            m_ui->resolutionComboBox->setCurrentIndex(0);
+        } else if (resolution == AudioInterface::BIT16) {
+            m_ui->resolutionComboBox->setCurrentIndex(1);
+        } else if (resolution == AudioInterface::BIT24) {
+            m_ui->resolutionComboBox->setCurrentIndex(2);
+        } else {
+            m_ui->resolutionComboBox->setCurrentIndex(3);
+        }
+        m_ui->connectAudioCheckBox->setChecked(
+            cliSettings->getConnectDefaultAudioPorts());
+        m_ui->realTimeCheckBox->setChecked(cliSettings->getUseRtUdpPriority());
+
+        m_ui->requireAuthCheckBox->setChecked(cliSettings->getUseAuthentication());
+        m_ui->authCheckBox->setChecked(cliSettings->getUseAuthentication());
+        m_ui->certEdit->setText(cliSettings->getCertFile());
+        m_ui->keyEdit->setText(cliSettings->getKeyFile());
+        m_ui->credsEdit->setText(cliSettings->getCredsFile());
+        m_ui->usernameEdit->setText(cliSettings->getUsername());
+        m_ui->passwordEdit->setText(cliSettings->getPassword());
+
+        settings.beginGroup(QStringLiteral("JitterBuffer"));
+        settings.setValue(QStringLiteral("JitterAnnounce"), true);
+        int bufferStrategy = cliSettings->getBufferStrategry();
```

Comment:
Typo: getBufferStrategry

Reply:
Damn it. That means it's wrong everywhere. Will change.